entrypoint/entrypoint.sh

```diff
@@ -9,7 +9,7 @@ ENDCOLOR='\033[0m' # No Color
 RED='\033[0;31m'
 
 # Enviornment variables: Observability operator test images 
-OPERATOR_BUNDLE=${OPERATOR_BUNDLE:="quay.io/gbernal/observability-operator-bundle:0.4.0-dev-tp"}
+OPERATOR_BUNDLE=${OPERATOR_BUNDLE:="quay.io/jezhu/observability-operator-bundle:0.4.0-release"}
```

Comment:
this should probably be the observability-ui org in quay

Reply:
Pushed `quay.io/openshift-observability-ui/observability-ui-operator-bundle:0.4.0` to the observability-ui org in quay.io [1] and scripts to use this as the default image. 

[1] https://quay.io/repository/openshift-observability-ui/observability-ui-operator-bundle?tab=tags

Reply:
I don't think we should try to keep the observability-ui org up to date. The observability-operator already mirrors to quay [here](https://quay.io/repository/rhobs/observability-operator-bundle). We can get the most up to date one using something simple like 

```
 podman image search --list-tags --no-trunc --limit 10000 quay.io/rhobs/observability-operator-bundle | tail -1
```

Granted as more commits go into the obseravility-operator this could become untenable, but in that case we should instead open a PR or ask the monitoring team if we could add the latest tag to that repo rather than trying to just mirror changes every once and a while

Reply:
This is a good point. It would be a bit a overhead to modify code (e.g. `main.go` > change `openshift.enable=true`) and rebuild in the RHOBS/ObO image for each release. 

As discussed in Slack DMs from October 1, 2024  7-9pm we will address this in a separate issue.  I will leave the image as is for now `quay.io/openshift-observability-ui/observability-ui-operator-bundle:0.4.0`.